.travis.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       name: "HTML"
       after_success: skip
     - script:
-        - julia --project=docs/pdf/ -e 'using Pkg; Pkg.instantiate()'
```

Comment:
@fredrikekre was there a reason we had a manifest committed for the PDF?

Reply:
Don't remember.